daemon/ucrednet.go

```diff
@@ -63,19 +66,31 @@ func ucrednetGetWithInterfaceImpl(remoteAddr string) (ucred *ucrednet, iface str
 			u.Uid = uint32(v)
 		}
 		u.Socket = subs[3]
-		if len(subs) == 6 {
-			iface = subs[5]
+		if len(subs[4]) > 0 {
+			ifaces = strings.Split(subs[5], "&")
 		}
 	}
 	if u.Pid == ucrednetNoProcess || u.Uid == ucrednetNobody {
-		return nil, "", errNoID
+		return nil, nil, errNoID
 	}
 
-	return u, iface, nil
+	return u, ifaces, nil
 }
 
+var ifaceRegexp = regexp.MustCompile(`iface=([^;]*)`)
```

Comment:
why not reuse the other regexp? also this can be at least anchored at the end?

Reply:
I didn't want to anchor it at the end so we'd have the potential to add other fields in the future. But if I annotate the `iface=` group with a named subgroup, we can just use that, and no need for this additional regexp.

Reply:
Also, it's so I could use `regexp.ReplaceAllString` to just replace the `iface=([^;]*);` part without touching the earlier parts of `remoteAddr`. I rewrote this now to use `raddrRegexp.FindStringSubmatchIndex(remoteAddr)`, which will allow extracting the string preceding the `iface=...` match, but I think it's a bit harder to read. Please let me know what you prefer.